### PR TITLE
Add PyGeoOGC package

### PR DESCRIPTION
**Pull request by cheginit**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cheginit** [comment]
@conda-forge/staged-recipes All tests have passed and it's ready for action! Thanks.

File: recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 0
```
Comment (scopatz) [inline recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml:14]: this is in a weird folder

Comment (cheginit) [reply recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml:14]: This was for the previous package that I removed and replaced with this new one. So it's deleted and is not in this commit.

Comment (cheginit) [reply recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml:14]: @scopatz Is there anything that I need to do?

Comment (scopatz) [reply recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml:14]: This ipynb folder shouldn't exist here at all.  Please just delete it

Comment (cheginit) [reply recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml]: I always have it in gitignore so I keep forgetting to check it. I just removed it.

File: recipes/pygeoogc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% set name = "pygeoogc" %}
```
Comment (scopatz) [inline recipes/pygeoogc/meta.yaml:1]: Please don't modify files outside of the recipe dir you are working on.  Maybe you need to pull from master?

File: recipes/hydrodata/meta.yaml [hunk 3]
```diff
@@ -1,67 +0,0 @@
-{% set name = "hydrodata" %}
```
Comment (scopatz) [inline recipes/hydrodata/meta.yaml:1]: Please don't modify files outside of the recipe dir you are working on. Maybe you need to pull from master?

File: recipes/pygeoogc/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/pygeoogc/meta.yaml:1]: wrong spot, sorry

File: recipes/hydrodata/meta.yaml (hunk 3 above)
Comment (cheginit) [reply recipes/hydrodata/meta.yaml:1]: This is already removed. I think you're looking at previous commits, not the latest one.
Do you mean that I close this PR an open a new one?

Comment (scopatz) [reply recipes/hydrodata/meta.yaml:1]: maybe just open a new PR

Comment (cheginit) [reply recipes/hydrodata/meta.yaml:1]: ok

Comment (cheginit) [reply recipes/hydrodata/meta.yaml:1]: Should I mention you there or wait for someone to pick it up?

**Pull request closed without merging**